snapcraft.yaml

```diff
@@ -0,0 +1,28 @@
+name: dss
+summary: a ML development and experimentation environment manager
+description: |
+     The DSS is a tool that allows users of workstations to deploy
```

Comment:
```suggestion
     The DSS is a tool that allows workstations users to deploy
```
?

Reply:
not really sure about it, if you don't think it's needed, you can keep it as is.